index.html

```diff
@@ -0,0 +1,141 @@
+        };
+      </pre>
+      <section>
+        <h3>The <dfn>visible</dfn> attribute</h3>
+        <p>The `visible` attribute is a boolean that determines if the window controls overlay has been rendered.</p>
+      </section>
+      <section>
+        <h3>The <dfn>getBoundingClientRect</dfn> method</h3>
+        <p>The `getBoundingClientRect()` method returns a DOMRect that represents the area under the window controls overlay. Web content SHOULD not be displayed beneath the overlay.</p>
+      </section>
+      <section>
+        <h3>The <dfn>ongeometrychange</dfn> attribute</h3>
+        <p>The `ongeometrychange` attribute is an event handler whose corresponding event handler event type is "change".</p>
+      </section>
+    </section>
+    <section>
+      <h2>
+        Addition of `window-controls-overlay` to the manifest
+      </h2>
+      <p>
+        To enable the WCO feature, a new value will be added to the `display-override` member of the manifest file. When the `window-controls-overlay` value is specified in the `display-override` the user agent SHOULD extend the client area to the title bar. 
+      </p>
+    </section>
+    <section>
+      <h2>CSS environmental variables</h2>
+      <aside class="issue">
+        <p>This is draft text for features that might belong in other specifications. This document might not be the final venue for some of the work specified in this draft.</p>
+      </aside>
+      <section>
+        <h3>Title bar area variables</h3>
+        <table dfn-type=value dfn-for="env()">
+          <tr>
+            <th>Name
+            <th>Value
+          <tr>
+            <td><dfn>`titlebar-area-x`</dfn>
+            <td><<length>>
+          <tr>
+            <td><dfn>`titlebar-area-y`</dfn>
+            <td><<length>>
+          <tr>
+            <td><dfn>`titlebar-area-width`</dfn>
+            <td><<length>>
+          <tr>
+            <td><dfn>`titlebar-area-height`</dfn>
+            <td><<length>>
+        </table>
+        <p>The title bar area variables are four [=environmental variables=] that define a rectangle by the width and height from a starting point in the viewport. This area corresponds to the area of the title bar, and allows to position dynamic Web content inside its bounds.</p>
+      </section>
+    </section>
+    <section>
+      <h2>Defining draggable sections</h2>
+      <p>To define which regions or elements in the title bar are draggable, this specification defines the CSS `-app-region` property.</p>
+    </section>
+    <section>
+      <h2>Algorithms</h2>
+      <section>
+        <h3>Update the overlay area information</h3>
+        <p>
+          The steps to <dfn>update the overlay information</dfn> of a
+          {{Document}} are as follows:
+        </p>
+        <ol class="algorithm">
+          <li>When the window or the controls overlay area resizes, the user agent updates the environmental variables to reflect the new available space in the titlebar.</li>
```

Comment:
-> "When the available title bar area resizes" or something along those lines. Technically, the WCO rect could change, but if the available rect (described by CSS env()s and JS APIs), then the `ongeometrychange` event won't fire.